locustfiles/prime_workflow.py

```diff
@@ -3,7 +3,8 @@
 
 from utils.hosts import MilMoveHostMixin, MilMoveDomain
 from utils.constants import PRIME_API_KEY, SUPPORT_API_KEY
-from tasks import PrimeWorkflowTasks
+from tasks.prime_hhg_workflow import PrimeWorkflowTasks
+from tasks.prime_endpoint_workflows import PrimeEndpointWorkflowsTasks
```

Comment:
Oh, this is super minor, but since you put the imports for these into the `tasks/__init__.py`, you can actually simplify this import to `from tasks import PrimeWorkflowTasks, PrimeEndpointWorkflowsTasks`